docs/DEV-TOOLS.md

```diff
@@ -0,0 +1,26 @@
+# Dev tools
+
+![dev view screenshot](assets/dev-view-screenshot.png)
+
+FlightCore features a hidden view that contains development features.
+
+It's targetted at both Northstar and FlightCore contributors. Among other things it contains buttons for unreleased features in FlightCore and tools to help with Northstar development.
+
+To activate it, spam click the FlightCore version number in the settings view at least 5 times. After that a new entry named _DEV_ should appear in the menubar.
+
+## Northstar
+
+### Pull request install
+
+The dev view offers a way to install pull request from the [NorthstarLauncher](https://github.com/R2Northstar/NorthstarLauncher) and [NorthstarMods](https://github.com/R2Northstar/NorthstarMods) repositories.
+
+Launcher pull requests overwrite `NorthstarLauncher.exe` and `Northstar.dll`.
+
+Mod pull requests install into a separate profile called `R2Northstar-PR-test-managed-folder`.
```

Comment:
You could mention the .bat file needed to actually use mod pull requests.

Reply:
Done in 560bd250694cc2e0f273edf28df2aa2deb53aa9b